public/js/components/FormFields/FormFieldCheckboxGroup.js

```diff
@@ -0,0 +1,68 @@
+import React, {PropTypes} from 'react';
+import ShowErrors from '../Utilities/ShowErrors';
+import { errorPropType } from '../../constants/errorPropType';
+import FormFieldCheckbox from './FormFieldCheckbox';
+
+export default class FormFieldCheckboxGroup extends React.Component {
+
+  static propTypes = {
+
+    fieldLabel: PropTypes.string.isRequired,
+    fieldName: PropTypes.string.isRequired,
+    fieldValue: PropTypes.array.isRequired,
+    checkValues: PropTypes.array.isRequired,
+    fieldErrors: PropTypes.arrayOf(errorPropType),
+    onUpdateField: PropTypes.func.isRequired
+  };
+
+  addOrRemoveValue = (fieldName) => {
+    let newFieldValue = [];
+
+    if(this.props.fieldValue.includes(fieldName)) {
+      newFieldValue = this.props.fieldValue.filter((value, currentIndex) => {
+        return currentIndex !== fieldName;
+      });
+    } else {
+      newFieldValue = this.props.fieldValue.concat(fieldName);
+    }
+
+    return newFieldValue;
+  }
+
+  isChecked = (checkValue) => {
+    return this.props.fieldValue.includes(checkValue);
+  }
+
+  renderCheckbox(fieldName, i) {
+    const updateFn = (newValue) => {
+      let newFieldValue = [];
+      if(newValue) {
+        newFieldValue = this.props.fieldValue.concat(fieldName);
```

Comment:
Yep, and that should happen as the true/false will _totally_ work. But I'll add a check just in case haha